internal/runners/prepare/prepare_darwin.go

```diff
@@ -19,13 +39,34 @@ func InstalledPreparedFiles(cfg autostart.Configurable) ([]string, error) {
 	if err != nil {
 		return nil, locale.WrapError(err, "err_tray_exec")
 	}
-	name, exec := constants.TrayAppName, trayExec
 
-	sc, err := autostart.New(name, exec, cfg).Path()
+	sc, err := autostart.New(autostart.Tray, trayExec, nil, cfg)
+	if err != nil {
+		return nil, locale.WrapError(err, "err_autostart_app")
+	}
+
+	path, err := sc.Path()
+	if err != nil {
+		multilog.Error("Failed to determine shortcut path for removal: %v", err)
+	} else if path != "" {
+		files = append(files, path)
+	}
+
+	svcExec, err := installation.ServiceExec()
+	if err != nil {
+		return nil, locale.WrapError(err, "err_svc_exec")
+	}
+
+	sc, err = autostart.New(autostart.Service, svcExec, []string{"start"}, cfg)
+	if err != nil {
+		return nil, locale.WrapError(err, "err_autostart_app")
+	}
+
+	path, err = sc.Path()
```

Comment:
This feels very similar if not identical to windows and linux. Does this need to be in prepareOS()? Curious if we can reduce the redundancy.